src/translation/translation_fr_FR.ts

```diff
@@ -3750,7 +3750,7 @@ Nous avons coupé votre canal et activé &quot;Me silencer&quot;. Veuillez d&apo
     <message>
         <location filename="../serverdlgbase.ui" line="75"/>
         <source>Directory</source>
-        <translation type="unfinished">Serveur annuaire</translation>
+        <translation>Serveur annuaire</translation>
```

Comment:
But here Directory has `Serveur` added.

Reply:
done.

Reply:
This appears unchanged.

Reply:
Oh, sorry - yes, you did do what I said, didn't you.  My mistake.  This needs to be "Annuaire", as elsewhere.

Reply:
Ok now. ;-)
No problem Peter.